.rubocop.yml

```diff
@@ -1,3 +1,6 @@
+inherit_gem:
+  chefstyle: config/chefstyle.yml
```

Comment:
Are you trying to run rubocop here on the cli and still get chefstyle?

Reply:
yes